demos/inpage/src/helpers/encodeUtils.ts

```diff
@@ -117,3 +117,25 @@ export function encodeBitStack(bits: boolean[]) {
 export function encodeBytes(bytes: string) {
   return hexJoin([encodeVLQ(BigInt(hexLen(bytes))), bytes]);
 }
+
+export function roundUpPseudoFloat(x: bigint) {
```

Comment:
Would be good to have unit tests for this function to ensure it works correctly on a range of values.

Reply:
Done: https://github.com/getwax/wax/blob/main/demos/inpage/tests/helpers/encodeUtils.test.ts